ios/Classes/SwiftMparticleFlutterSdkPlugin.swift

```diff
@@ -12,7 +12,7 @@ public class SwiftMparticleFlutterSdkPlugin: NSObject, FlutterPlugin {
   public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
     switch call.method {
     case "isInitialized":
-        result(MParticle.sharedInstance() != nil)
+        result(true)
```

Comment:
I think we want to revert this change. We want to know if there's a shared instance or not, and not always return true.

Reply:
The reason @einsteinx2 changed this is because of the singleton pattern we will always have an instance and it creates a warning to have the code from before.

Reply:
Yes exactly, the instance is always there so for iOS it's actually always initialized. That previous line caused a warning since it's always evaluated to true, so I just made it return true.

Reply:
So, I think this was probably bad logic to begin with, but `isInitialized` _should_ be returning a boolean based on whether the SDK has been started (aka has `start()` been called and are we good to log events and such)

Reply:
OK got it. I think we were confirming whether or not the instance was available. Updated to check if initialized.